lib/datadog/tracing/contrib/pg/integration.rb

```diff
@@ -0,0 +1,43 @@
+# typed: false
+
+require 'datadog/tracing/contrib/integration'
+require 'datadog/tracing/contrib/pg/configuration/settings'
+require 'datadog/tracing/contrib/pg/patcher'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module Pg
+        # Description of pg integration
+        class Integration
+          include Contrib::Integration
+
+          MINIMUM_VERSION = Gem::Version.new('1.1.4')
```

Comment:
Any particular reason for choosing `1.1.4`?
Ideally we'd have no restrictions on what version of the gem we support (meaning we support all versions), but that's rarely the case, especially for super early versions.

I recommend support the oldest version that does not cause your tests to fail: https://rubygems.org/gems/pg/versions

We can also add `if` conditions in our code to support super old versions, but that more of a judgment call that we can make, and normally only applies if the library in question has changed a lot recently, and the latest version does not represent the majority of user base.